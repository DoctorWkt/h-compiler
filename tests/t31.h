char str1[]="Hello";
int ilist1[]= { 1, 2, 3};
int empty1[4];
void main(int argc, char *argv)
{
  char str2[]="abc";			// Local arrays now!
  int ilist2[]= {1, 4, 2};
  int empty1[4];
  int x;

  x=str1[0]; putchar(x); putchar('\n');
  x=str2[0]; putchar(x); putchar('\n');

  x= ilist1[2]; printn(x);
  x= ilist2[2]; printn(x);

  empty1[2]= 2 + 3; printn(empty1[2]);
}
