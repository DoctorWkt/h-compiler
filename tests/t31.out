H
a
000003
000002
000005
